c10/core/WrapDimMinimal.cpp

```diff
@@ -3,10 +3,17 @@
 namespace c10 {
 namespace detail {
 
-int64_t maybe_wrap_dim_slow(
+template int64_t maybe_wrap_dim_slow(
     int64_t dim,
     int64_t dim_post_expr,
-    bool wrap_scalar) {
+    bool wrap_scalar);
+template SymInt maybe_wrap_dim_slow(
+    SymInt dim,
+    SymInt dim_post_expr,
+    bool wrap_scalar);
```

Comment:
Hmm, I think you need to annotate these with `C10_API`, and that's why you got linker errors when you tried to template